apps/els_lsp/src/els_diagnostics.erl

```diff
@@ -64,6 +64,8 @@ available_diagnostics() ->
   , <<"elvis">>
   , <<"unused_includes">>
   , <<"unused_macros">>
+  , <<"referl_unused_macros">>
```

Comment:
In my mind, it could make more sense to have `<<"refactorerl">>` as a single diagnostics backend, instead of one per feature. This would also help removing some of the existing duplication between the two newly introduced modules. What do you think?